tests/compile-fail/fs/close_stderr.rs

```diff
@@ -0,0 +1,14 @@
+// ignore-windows: No libc on Windows
+// compile-flags: -Zmiri-disable-isolation
+
+// https://github.com/rust-lang/rust/issues/40032
+
+#![feature(rustc_private)]
+
+extern crate libc;
+
+fn main() {
+    unsafe {
+        libc::close(2); //~ ERROR stdin/stdout/stderr cannot be closed
```

Comment:
IMO having the stdout test is sufficient, no need to be exhaustive here.

Reply:
Removed this test